v2/app/collegevine-hub/CVHubViewer.jsx

```diff
@@ -1522,6 +1535,10 @@ var PivotViewer = (Pivot.PivotViewer = function (
           }
         }
 
+        // if an item is selected, disable focus on hover
+        if (!anyItemHovered && oldHoveredItem){
+          self.trigger("itemBlur")
+        }
```

Comment:
Ditto: Can we use `hoveredItem`?

```suggestion
        if (!hovereddItem && oldHoveredItem){
          self.trigger("itemBlur")
        }
```